distribution/pom.xml

```diff
@@ -394,6 +394,11 @@
             <artifactId>hazelcast-3-connector-common</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.hazelcast.jet</groupId>
```

Comment:
Do we want to keep a separate group for jet-related artifects?

Reply:
Good point, changed to plain `com.hazelcast` for this module.